packages/backend/src/vscode-youi-events.ts

```diff
@@ -29,6 +29,11 @@ class YoUiAppWizard extends AppWizard {
   public showProgress(message?: string): void {
     this.events.showProgress(message);
   }
+
+  // Allows generators to update the App Wizard title
+  public setHeaderTitle(title: string, additionalInfo?: string): void {
+    this.events.setAppWizardHeaderTitle(title, additionalInfo);
+  }
 }
 
 export class VSCodeYouiEvents implements YouiEvents {
```

Comment:
seems like you need to update the YouiEvents interface accordingly - declare the 'setAppWizardHeaderTitle' method there

Reply:
There is no need since we will never use that implementation. Only the app wizard interface requires this API function. Adding to `YouiEvents` results in unecessary updates to `ServerYouiEvents` also. We will never use that.